skin.css

```diff
@@ -2,12 +2,65 @@
     background-color: #ddd;
 }
 
-button.btn {
+.btn {
+  width:170px
   border-radius: 3px;
-  border-style: solid;
-  border-width: 1px;
+  background-color:white;
   font-weight: normal;
-  line-height: 1.15;
-  text-align: center;
-  vertical-align: middle;
+  font-family:sans-serif;
+  padding: 15px;
+  border: 1px; solid lightgrey
+
+
+}
+
+
+  .btn--default{
+   border-style:lightgrey
+    
+       } 
+.btn--primary{
+  background-color:blue;
+  color:white;
+  }
+  .btn--secondary{
+    background-color:gainsboro;
+    color:grey;
+    
+  }
+  .btn--large{
+    width:250px;
+    padding:20px;
+    color:blue;
+  }
+  .btn--extra-large{
+    width:350px;
+    padding:20px;
+    background-color: white;
+    color: blue;
+    font-size:20px;
+
+  }
+  nav.ul{
+    float: left;
+    
+  }
+  ul li{

```

Comment:
This `ul li` code will apply to any `ul` elements on the page, even when they are not wrapped with an element with a class of `nav`. To explain better, both of examples below will end up having these styles.

```
<div class="nav">
  <ul>
    <li>Menu item</li>
    <li>Menu item</li>
  </ul>
</div>
```

```
<div class="something-else">
  <ul>
    <li>Menu item</li>
    <li>Menu item</li>
  </ul>
</div>
```

In order to get these styles to be applied only to `nav`, your rule will need to be `nav ul li { ... }`. The same thing applies to line `57`.

[More reading on `descendant selectors` >](https://www.sitepoint.com/web-foundations/descendant-selector-css-selector/)